py/torch_tensorrt/dynamo/_defaults.py

```diff
@@ -27,7 +27,7 @@
 REQUIRE_FULL_COMPILATION = False
 DRYRUN = False
 HARDWARE_COMPATIBLE = False
-SUPPORTED_KERNEL_PRECISIONS = {dtype.f32, dtype.f16, dtype.i8}
+SUPPORTED_KERNEL_PRECISIONS = {dtype.f32, dtype.f16, dtype.i8, dtype.f8}
```

Comment:
Missing bf16 here, can you just use a cherrypick of main?

Reply:
https://github.com/pytorch/TensorRT/pull/2845

Reply:
This will be added when we merge this https://github.com/pytorch/TensorRT/pull/2845 right ?